concent_api/concent_api/settings/testing.py

```diff
@@ -77,6 +77,8 @@
 
 CUSTOM_PROTOCOL_TIMES = True
 
+CONCENT_ETHEREUM_PUBLIC_KEY = '04efb99d9860f4dec4cb548a5722c27e9ef58e37fbab9719c5b33d55c216db49311221a01f638ce5f255875b194e0acaa58b19a89d2e56a864427298f826a7f887'
+
```

Comment:
This should only be hard-coded in `development.py`. Tests should set it themselves, just like they do for the message keys.

And in `development.py` - only hard-code if if there's some universal key usable out of the box.

Reply:
@cameel It is generated from `PRIVATE_KEY` from https://theethereum.wiki/w/index.php/Accounts,_Addresses,_Public_And_Private_Keys,_And_Tokens

Is that ok to hardcode it in development ?

Reply:
The idea is that you should be able to use settings from `development.py` out of the box in most cases, without having to modify them.

Reply:
I think it would be better to fail, and set this value in `local_settings.py` if needed.

Reply:
In that case we don't want it in `development.py` either.

At most we might add a `manage.py check` for this setting with a hint about where to get a real key from.